packages/devextreme/testing/tests/DevExpress.ui.events/remove.tests.js

```diff
@@ -22,6 +22,8 @@ QUnit.test('dxremove event should be fired on element removing', function(assert
 });
 
 QUnit.test('dxremove event should be fired for nested removing element', function(assert) {
+    assert.expect(2);
```

Comment:
I accidentally came across this test that seemingly lacks an expected number of assertions... Irrelevant to the main PR goal, so if you want, I can remove it.